addons/web/static/src/views/kanban/kanban_controller.js

```diff
@@ -88,8 +88,17 @@ export class KanbanController extends Component {
                 return super.load(...arguments);
             }
 
-            async _webReadGroup() {
-                const result = await super._webReadGroup(...arguments);
+            async _webReadGroup(config, orderBy) {
+                const activeFields = Object.fromEntries(
+                    Object.entries(config.activeFields).filter(
+                        ([fieldName, values]) => config.activeAggregates.includes(fieldName)
+                    )
+                );
+                const result = await super._webReadGroup({
+                    ...config,
+                    // remove fields because we don't use any aggregate in the kanban views
```

Comment:
this override exists for the sample data feature, it's a bit weird to put optimizations for aggregate fields there.